### PR TITLE
Warehouse-specific configuration improvements

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Added some missing details for  GCP bucket (data lake), Snowflake and Azure Synapse
https://www.notion.so/rudderstacks/Issues-in-warehouse-specific-configurations-in-docs-9151ac68e8764e20a8c8c64a85a69e92

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by achettyiitr — APPROVED** [review]
LGTM

File: docs/data-warehouse-integrations/azure-synapse.mdx [hunk 1]
```diff
@@ -22,7 +22,7 @@ Find the open-source transformer code for this destination in the <a href="https
 
 ## Setting user permissions in Azure Synapse
 
-To set up an Azure Synapse Analytics user, you need an Azure subscription for the [SQL Data Warehouse](https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/create-data-warehouse-portal).
+To set up an Azure Synapse Analytics user, you require an Azure subscription for the [SQL Data Warehouse](https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/create-data-warehouse-portal). Additionally, you need to create a schema (dedicated SQL pool) in Azure Synapse.
```
Comment (ameypv-rudder) [inline docs/data-warehouse-integrations/azure-synapse.mdx:25]: Should we add a reference link to create a schema in a dedicated SQL pool in Synapse? 

https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/create-data-warehouse-portal -- is this the correct reference? Or would [this link](https://docs.microsoft.com/en-us/sql/relational-databases/security/authentication-access/create-a-database-schema?view=sql-server-ver16) be the correct reference? @achettyiitr @saramashfej can you please help?

Comment (achettyiitr) [reply docs/data-warehouse-integrations/azure-synapse.mdx:25]: Let's move the link provided with SQL Data Warehouse to dedicated SQL pool.

**Review by ameypv-rudder — APPROVED** [review]

File: docs/data-warehouse-integrations/azure-synapse.mdx (hunk 1 above)
Comment (saramashfej) [reply docs/data-warehouse-integrations/azure-synapse.mdx]: This link points to creating a Dedicated SQL pool which should be added as a pre-req for this guide. Can we add a line in the beginning that says "You will need to have a Dedicated SQL pool created in Azure Synapse as a pre-requisite for this guide, if you do not have one you can create it using this link https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/create-data-warehouse-portal" 
"You will need to create a user defined schema on your dedicated SQL pool, please run this SQL code on your pool:
`CREATE SCHEMA [your-schema-name]; 
GO`

Comment (saramashfej) [reply docs/data-warehouse-integrations/azure-synapse.mdx]: Can we then remove the part in [https://deploy-preview-600--rudderstack-docs-staging.netlify.app/data-warehouse-integrations/azure-synapse/#connection-settings] Connection settings that says, "If you don't specify any namespace, RudderStack will set this to the source name, by default."

**Pull request merged**